yarn/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG - yarn
 
+## 1.17.2 / 2021-02-10
+
+* [Added] Add config option to disable node metric collection
+
```

Comment:
```suggestion
```
The changelog is automatically generated when we release integration wheels. No need to update here